doc/import-commitments.md

```diff
@@ -17,13 +17,15 @@ RODA-ID,100000,2023-01-02
 
 The header fields are case sensitive.
 
+The import script does not accept 'Commitment value' in the form of a string. If the csv has any values provided as strings, the string quotes around the number must be removed before running.
+
 A RODA user email address to be used to log the user who is recorded in the
 history log as importing the commitments.
 
 No authorisation is performed, as long as the RODA identifier exists and does
 not already have a commitment, the importer will set one.
 
-Copy the csv to production, [see details here](./importing-new-delivery-partner-data.md#import-activity-data-in-production)
+Copy the csv to production, [see details here](./uploading-and-dowloading-files.md)
```

Comment:
typo in `dow[n]loading-files` in the link here.

Reply:
The i[n]dentifier stole it